response_operations_ui/static/js/validate-ci.js

```diff
@@ -1,40 +1,47 @@
-function checkCI(file){
+(function(window) {
+    function checkCI(file) {
+        const type = file.type;
+        const errorPanel = document.getElementById('ciFileErrorPanel');
+        const errorPanelBody = document.getElementById('ciFileErrorPanelBody');
+
+        if ( type === 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet' ) {
+            errorPanel.classList.remove('panel');
+            errorPanel.classList.remove('panel--simple');
+            errorPanel.classList.remove('panel--error');
+
+            errorPanelBody.classList.remove('panel__body');
+            paragraphsToRemoveClassFrom = Array.prototype.slice.call(errorPanelBody.querySelectorAll('p'));
+
+            paragraphsToRemoveClassFrom.forEach(function(element) {
+                element.classList.remove('hidden');
+            });
+
+            document.getElementById('btn-load-ci').classList.remove('unready');
+        } else {
+            errorPanel.classList.add('panel');
+            errorPanel.classList.add('panel--simple');
+            errorPanel.classList.add('panel--error');
+
+            errorPanelBody.classList.add('panel__body');
+            paragraphsToAddClassTo = Array.prototype.slice.call(errorPanelBody.querySelectorAll('p'));
```

Comment:
Any value in putting a comment like 
`Equivalent to $("#ciFileErrorPanelBody p").addClass("hidden"); in jquery` ?

Reply:
Probably arguable to make reference to jQuery specifically - but I can see value to commenting this part, given that it uses a slightly ugly method to convert the `NodeList` to a proper `Array`.

Change on the way.